tests/MongoDB.Driver.Tests/Specifications/transactions/TransactionTestRunner.cs

```diff
@@ -295,24 +356,50 @@ private ClientSessionOptions ParseSessionOptions(BsonDocument test, string sessi
             return options;
         }
 
-        private FailPoint ConfigureFailPoint(BsonDocument test)
+        private FailPoint ConfigureFailPoint(IServer server, BsonValue failPoint)
+        {
+            var session = NoCoreSession.NewHandle();
+            var command = failPoint.AsBsonDocument;
+            return FailPoint.Configure(server, session, command);
+
+        }
```

Comment:
I would get rid of this ConfigureFailPoint method, which would no longer be called after a change suggested below.